src/dog.sol

```diff
@@ -149,6 +149,77 @@ contract Dog {
     }
 
     // --- CDP Liquidation: all bark and no bite ---
+    //
+    // This function computes `tab`, the target amount of DAI to be raised in
+    // an auction. In the general case, it is the product of `dart`, the
+    // collateral's accumulated rate, and the collateral's `chop` value.
+    //
+    //     tab = dart * ilk.rate * ilk.chop
+    //
+    // The `dart` value stands for `delta art` and it is equal to or less than
+    // the vault's normalized debt (`art`) value:
+    //
+    //     dart <= vault.art
+    //
+    // When a liquidation is successfully started, its `tab` value is added to
+    // the collateral's `dirt` value:
+    //
+    //     ilk.dirt' = ilk.dirt + tab
+    //
+    // To prevent too many liquidations from being triggered at once, the `tab`
+    // value of new liquidations added to the collateral's `dirt` shouldn't
+    // surpass the collateral's `hole` value:
+    //
+    //     tab + ilk.dirt <= ilk.hole
```

Comment:
I know this will be taken care below, and it's probably difficult to express the whole thing differently, but if one reads just this equation will be misleading.

Reply:
I'm handling the dusty case separately, that's why `dust` isn't included there. But I agree it's confusing: it'd be interesting to be able to handle both cases with a single formula, but it doesn't seem to be an easy task.

Reply:
Left a comment in the general dialog to simplify `bark`. Let's continue the discussion there.